src/bp-core/bp-core-functions.php

```diff
@@ -4936,12 +4936,7 @@ function bp_core_parse_url( $url ) {
 	} else {
 
 		// safely get URL and response body.
-		$response = wp_safe_remote_get(
-			$url,
-			array(
-				'user-agent' => '', // Default value being blocked by Cloudflare
-			)
-		);
+		$response = wp_safe_remote_get( $url );
```

Comment:
@sadrul Please test after removing the code the original issue was fixed as per the PR: https://github.com/buddyboss/buddyboss-platform/pull/1520

Reply:
@KartikSuthar I tested after removing that code and found it working